frontend/src/modules/onboarding/pages/WorkspaceInvitationPage/WorkspaceInvitationPage.jsx

```diff
@@ -97,7 +97,7 @@ const WorkspaceInvitationPage = (props) => {
               disabled={true}
               readOnly="true"
               name="email"
-              dataCy="email-input"
+              dataCy="email"
```

Comment:
Create a PR for front-end changes

Reply:
Will this same in Frontend - No changes for now